cfme/intelligence/reports/reports.py

```diff
@@ -558,6 +532,10 @@ def delete(self, cancel=False):
             # TODO Doesn't work due to this BZ https://bugzilla.redhat.com/show_bug.cgi?id=1489387
             # view.flash.assert_message("Successfully deleted Saved Report from the CFME Database")
 
+    @property
+    def tree_path(self):
+        return self.report.tree_path + [self.datetime_in_tree]
```

Comment:
Optional: a `.append()` may be more explicit here

Reply:
Yep, but I was trying to maintain the pattern that has been used at other places where `tree_path` has been defined.